instrumentation/jaxrs/jaxrs-2.0/jaxrs-2.0-resteasy-3.0/javaagent/build.gradle.kts

```diff
@@ -40,7 +40,7 @@ dependencies {
   testLibrary("org.jboss.resteasy:resteasy-undertow:3.0.4.Final") {
     exclude("org.jboss.resteasy", "resteasy-client")
   }
-  testLibrary("io.undertow:undertow-servlet:1.0.0.Final")
+  testLibrary("io.undertow:undertow-servlet:1.4.28.Final")
```

Comment:
> Undertow 1.0 had a bug where it thrown NPE on getHeaders()

😱

Reply:
```java
    public Collection<String> getHeaders(String name) {
        return new ArrayList(this.exchange.getResponseHeaders().get(name));
    }
```

Guess what happens when `this.exchange.getResponseHeaders()` returns null because there are no headers in the response yet 🙈